### ISSUE TITLE
Visualization:  graphs not loading since 12/13/2019.

### ISSUE THREAD

**Issue by PattyHaug** [ODM2/ODM2DataSharingPortal#448]
The last day the visualization for selected parameters was working was 12/12/2019.  This feature is not loading and graphing selected parameters.  This is day three of this issue

**Comment by dbressler75** [comment]
@aufdenkampe, @SRGDamia1, Data are making it to monitor my watershed (they can be downloaded and current measurement shows up below sparklines) but no sparklines are working and none of the time series analyst tabs are working

**Comment by aufdenkampe** [comment]
Thanks for the heads up that this has happened again.
We became aware of this issue just before the weekend, and an attempt at a quick fix didn’t work. We're working on it now.

This is the same issue we’ve had to deal with for years, where the database connections to the sparkling and TSA visualization interfaces get messed up. The long-term fix unfortunately requires an architectural overhaul. See https://github.com/ODM2/ODM2DataSharingPortal/issues/434

Fortunately, the data are safe.

**Comment by tslawecki** [comment]
A first look shows that the influx database is up and running ...  following example query works:

`curl -G http://dbs.monitormywatershed.org:8086/query -u [user]:[pass] --data-urlencode "q=SHOW DATABASES"`

This one works too, but flags (in Chrome) as not secure:

https://dbs.monitormywatershed.org/query?u=[xxxx]&p=[yyyy]&db=envirodiy&q=SELECT%20time,%20DataValue::field,%20UTCOffset::field%20FROM%20uuid_6593b63a_349d_45e9_80c4_46996937ac49%20WHERE%20time%20%3E=2018-12-11

Checking on the server, the SSL certificate looks good:

```
server_admin@wpfdbsprod:/var/log$ sudo certbot certificates
[sudo] password for server_admin:
Saving debug log to /var/log/letsencrypt/letsencrypt.log

-------------------------------------------------------------------------------
Found the following certs:
  Certificate Name: dbs.monitormywatershed.org
    Domains: dbs.monitormywatershed.org
    Expiry Date: 2020-02-11 13:39:00+00:00 (VALID: 56 days)
    Certificate Path: /etc/letsencrypt/live/dbs.monitormywatershed.org/fullchain.pem
    Private Key Path: /etc/letsencrypt/live/dbs.monitormywatershed.org/privkey.pem
-------------------------------------------------------------------------------

```
However, an external [check](https://www.whynopadlock.com/results/6407e15d-3260-4cff-8b40-22b8d93cee1d) suggests otherwise, and indicates that the certificate expired Friday:

 _Expiration Date Your SSL certificate is expired! (Expired on: 2019-12-13) You will want to renew your SSL certificate as soon as possible!_

I renewed the SSL certificate, but it was still flagged as insecure. Digging further, I found some threads that suggested a web server restart might be needed, so I restarted nginx on dbs.monitormywatershed,org. This seemed to take care of the issue - please confirm.

Bottom line - when we get reports of TSA issues, let's check SSL status from *outside* the data server, as the internal status may not be accurate.

**Comment by aufdenkampe** [comment]
@tslawecki, thanks! That worked!
